tests/output/output_test.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from privacy_evaluator.metrics.basics import *
+from privacy_evaluator.output.user_output import UserOutput
+
+
+def test_output_function():
```

Comment:
Is it on purpose that nothing is checked in this test? Could you again explain what your thoughts were? In my opinion, you should assert that the histogram has the correct number of outputs per class.

Reply:
It was just checking it no errors ar thrown. But i refined the test now